vertx-context/src/main/java/io/smallrye/common/vertx/ContextLocals.java

```diff
@@ -0,0 +1,85 @@
+package io.smallrye.common.vertx;
```

Comment:
👍  I think it is not necessary to add accessor for localContextData

Reply:
Do you mean the method returning the map?

Reply:
Yes, but I don't think that it is needed.

Reply:
Agreed, I was thinking about it. 
We will see if we have a use case (it's easy to add, not to remove)